bitnami/phpmyadmin/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: phpmyadmin
 sources:
   - https://github.com/bitnami/bitnami-docker-phpmyadmin
   - https://www.phpmyadmin.net/
-version: 8.4.0
+version: 8.4.1
```

Comment:
Could you bump the minor version instead? It is adding a non-breaking feature

Reply:
No problem, I've bumped the minor version for this PR.